### ISSUE TITLE
Bug: Merge Build Doesn't import Form when not in already built DB.

### ISSUE THREAD

**Issue by hecon5** [joyfullservice/msaccess-vcs-addin#471]
Related to #469, I noticed that if a `Form` does not exist in the database currently, the form doesn't seem to import if it's in the source file but not the internally built database when doing a merge build. 

I'm not sure if that's the expected behavior, but it's something I thought I'd inquire about.

**Comment by joyfullservice** [comment]
No, the expected behavior is that a new source file for an object that doesn't exist in the database should be merged into the database. In theory, you should be able to drop in some source files into the source folder, and have them load into the database when you merge.

**Comment by hecon5** [comment]
Hmm, interesting. I'll dig into this a bit and see if I can force it. A part of me wonders if there's something else going on there.

**Comment by hecon5** [comment]
I tested this again, here's some long-winded info.

TL/DR: It looks like (w/ split forms) some of the file is incorrectly removed when exporting, and some is left, leaving a partially exported change. This doesn't seem to matter if it is a split form db or not. A deleted form will not re-import upon merge build. 

I tried this with a non-split form database (the same code base, even) that is pre-split (forked a variant to test before we split the forms), and the results are the same: deleted forms do not import again until they get re-restored, and it looks like when forms/reports are split, only part of the form / report is correctly deleted. 

Steps:
1. Delete a form (`frmCalendar` for consistency's sake). 
2. Do merge build (form is not re-imported).
3. Close / reopen db.
4. Do another merge build (The form is not imported)

If I do an export, it will remove the form from the the index, but a merge build does not re-import the file. I closed/reopened the db numerous times, and did a merge-build, but it will not re-import. It also says no changes found, too. 

When I deleted the form and did an export, it only removed the `.bas` file. The `.cls` file was left (and detected as a change). This happens even if I select "skip" (because `frmCalendar.bas` was deleted and wasn't detected as a difference). The external file is deleted regardless. If I restore the `frmCalendar.bas` file and do another export, THEN both `.bas` and `.cls` files are detected (and if I select skip then they're left this time).

After deleting form:
![image](https://github.com/joyfullservice/msaccess-vcs-addin/assets/54177882/8c857d09-fb29-4f91-8ef3-75f350e4ea84)

Merge build result immediately after deletion: 
![image](https://github.com/joyfullservice/msaccess-vcs-addin/assets/54177882/3690cef4-8885-40b7-b3d0-089886375f8b)

Export result:
![image](https://github.com/joyfullservice/msaccess-vcs-addin/assets/54177882/28d5f167-95fe-4511-9eda-32b01d99fdf6)
![image](https://github.com/joyfullservice/msaccess-vcs-addin/assets/54177882/2979f3a3-23f7-47ef-8b92-a6cad27babc8)

Import Result: 
![image](https://github.com/joyfullservice/msaccess-vcs-addin/assets/54177882/155b0ca3-8405-4e06-b794-6ab57c0c4472)

After restoring `frmCalendar.bas` (and doing another export):
![image](https://github.com/joyfullservice/msaccess-vcs-addin/assets/54177882/77e8e540-18d5-4599-982f-e05b71a8b971)

Do another import and NOW the form is correctly imported:
![image](https://github.com/joyfullservice/msaccess-vcs-addin/assets/54177882/114fb2be-facd-4988-9c30-6989d883aaaa)
![image](https://github.com/joyfullservice/msaccess-vcs-addin/assets/54177882/2d7237a2-6038-4813-8c58-dcc33bb144a2)

**Comment by joyfullservice** [comment]
This should be working correctly as of 4.0.30. Let me know if you have any issues with merge builds after updating.
